lib/resources/Recipients.js

```diff
@@ -1,48 +1,8 @@
 'use strict';
 
 const StripeResource = require('../StripeResource');
-const stripeMethod = StripeResource.method;
 
 module.exports = StripeResource.extend({
   path: 'recipients',
-  includeBasic: [
-    'create',
-    'del',
-    'list',
-    'retrieve',
-    'update',
-    'setMetadata',
-    'getMetadata',
-  ],
-
-  createCard: stripeMethod({
-    method: 'POST',
-    path: '/{id}/cards',
-    urlParams: ['id'],
-  }),
-
-  listCards: stripeMethod({
```

Comment:
How do you list cards if not through this? Sorry I don't remember what we discussed already :goldfish:

Reply:
It's no longer in the spec, so... you don't?

Reply:
After consideration, these methods should be in the spec, so I'll re-add them.